src/main/java/com/autotune/analyzer/services/BulkService.java

```diff
@@ -0,0 +1,126 @@
+    private ExecutorService executorService = Executors.newFixedThreadPool(10);
+    private Map<String, BulkJobStatus> jobStatusMap = new ConcurrentHashMap<>();
+
+    @Override
+    public void init(ServletConfig config) throws ServletException {
+        super.init(config);
+    }
+
+    /**
+     * @param req
+     * @param resp
+     * @throws ServletException
+     * @throws IOException
+     */
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String jobID = req.getParameter("jobID");
+        BulkJobStatus jobStatus = jobStatusMap.get(jobID);
+
+        if (jobStatus == null) {
+            resp.setStatus(HttpServletResponse.SC_NOT_FOUND);
+            resp.getWriter().write("{\"error\":\"Job not found\"}");
+        } else {
+            try {
+                resp.setContentType("application/json");
+                resp.setCharacterEncoding("UTF-8");
+                // Return the JSON representation of the JobStatus object
+                ObjectMapper objectMapper = new ObjectMapper();
+                String jsonResponse = objectMapper.writeValueAsString(jobStatus);
+                resp.getWriter().write(jsonResponse);
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    /**
+     * @param request
+     * @param response
+     * @throws ServletException
+     * @throws IOException
+     */
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        // Set response type
+        response.setContentType("application/json");
+
+        // Create ObjectMapper instance
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        // Read the request payload and map to RequestPayload class
+        BulkInput payload = objectMapper.readValue(request.getInputStream(), BulkInput.class);
+
+        // Generate a unique jobID
+        String jobID = UUID.randomUUID().toString();
+        BulkJobStatus.Data data = new BulkJobStatus.Data(
+                new BulkJobStatus.Experiments(new ArrayList<>(), new ArrayList<>()),
+                new BulkJobStatus.Recommendations(0, 0, new BulkJobStatus.RecommendationData(
+                        new ArrayList<>(),
+                        new ArrayList<>(),
+                        new ArrayList<>(),
+                        new ArrayList<>()
+                ))
+        );
+        jobStatusMap.put(jobID, new BulkJobStatus(jobID, "IN_PROGRESS", 0, data, Instant.now()));
+        // Submit the job to be processed asynchronously
+        // TODO example : executorService.submit(new BulkJobManager(jobID, jobStatusMap, payload));
+
+        // Just sending a simple success response back
+        // Return the jobID to the user
+        response.setContentType("application/json");
+        response.getWriter().write("{\"jobID\":\"" + jobID + "\"}");
```

Comment:
Lets keep it consistent with the other APIs

Reply:
done